cli/tests/integration/jsr_tests.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_lockfile::Lockfile;
+use test_util as util;
+use util::env_vars_for_jsr_tests;
+use util::TestContextBuilder;
+
+itest!(no_module_graph_run {
+  args: "run jsr/no_module_graph/main.ts",
```

Comment:
I'm not requiring an `--unstable` flag here at the moment because this feature is currently not usable so it will error every time anyway.